src/rdkafka_cgrp.c

```diff
@@ -2929,7 +2929,10 @@ void rd_kafka_cgrp_serve (rd_kafka_cgrp_t *rkcg) {
         case RD_KAFKA_CGRP_STATE_UP:
 		/* Move any ops awaiting the coordinator to the ops queue
 		 * for reprocessing. */
-		rd_kafka_q_concat(rkcg->rkcg_ops, rkcg->rkcg_wait_coord_q);
+			if (rd_kafka_q_len(rkcg->rkcg_wait_coord_q))
```

Comment:
I think this should be fixed in q_concat() instead, returning immediately if srcq is empty.